### DISCUSSION TITLE
Github security alert digest

### DISCUSSION THREAD

**Discussion by potapkin-pavel** [community/community#147353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Today I received the following email 
![image](https://github.com/user-attachments/assets/a6ad4983-ce76-410d-b3cc-b5c0fe1b4578)
and I have a few questions,
Why am I getting this? What's happened recently that github starts sending this spam? Did I change something? How to turn it off?

**Comment by davevad93** [comment]
Hi @potapkin-pavel ,

To disable `Dependabot` alerts by email just go to your [notification settings](https://github.com/settings/notifications), scroll down a bit to `Dependabot Alerts: New vulnerabilities` and uncheck "**Email**".

<img width="692" alt="00" src="https://github.com/user-attachments/assets/722902f2-70a1-4322-b36b-25b097120426" />

You can also disable all weekly email digests by selecting "**Don't send**" under `Email weekly digest`.

<img width="704" alt="0000" src="https://github.com/user-attachments/assets/43e0f8b4-9a8e-452a-b672-2d709eb343a2" />

**Comment by potapkin-pavel** [comment]
Hey @davevad93, great thank you for your answer. I just don't understand why I started getting this kind of email for a project I hadn't touched in years, I just don't understand why my mailbox is suddenly being treated like a dumpster, and I was desperately hoping that someone from github would just shed some light on this.

**Reply by davevad93** [reply]
Hey @potapkin-pavel you're welcome, as I'm not from GitHub I can't help you shed some light on this issue, you could try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and ask them.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @potapkin-pavel!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
